tests/hazmat/backends/test_openssl.py

```diff
@@ -282,6 +269,20 @@ def test_activate_builtin_random_already_active(self):
         e = backend._lib.ENGINE_get_default_RAND()
         assert e == backend._ffi.NULL
 
+    def test_osrandom_engine_implementation(self):
+        name = backend.osrandom_engine_implementation()
+        assert name in ['/dev/urandom', 'CCRandomGenerateBytes',
+                        'CryptGenRandom', 'getentropy', 'getrandom']
+        if sys.platform.startswith('linux'):
+            assert name in ['getrandom', '/dev/urandom']
+        if sys.platform == 'darwin':
+            assert name in ['CCRandomGenerateBytes', '/dev/urandom']
+        # TODO: enable when CI can handle coverage.py for these platforms
+        # if 'bsd' in sys.platform:
+        #     assert name in ['getentropy', '/dev/urandom']
+        # if sys.platform == 'win32':
```

Comment:
This branch can be uncommented now (I updated our jenkins configs so we now merge coverage across both jenkins and travis)